source/module_hsolver/diago_dav_subspace.h

```diff
@@ -31,6 +31,8 @@ class Diago_DavSubspace : public DiagH<T, Device>
 
     static int PW_DIAG_NDIM;
 
+    static double dav_large_thr;
```

Comment:
How about initialize the variable?

Reply:
look line143: double Diago_DavSubspace<Real, Device>::dav_large_thr = 1e-5;